source/air/src/ast.rs

```diff
@@ -49,6 +51,14 @@ pub enum BinaryOp {
     Gt,
     EuclideanDiv,
     EuclideanMod,
+
+    BitXor,
```

Comment:
Is `BitXor` being especially highlighted with extra space, or is this just an unintended artifact?  Same thing seems to happen in a few places below, e.g., in `source/air/src/printer.rs`.

Reply:
Oh it was somewhat intended to highlight the added operations due to this change. We could remove it if needed.